src/lcm/function_representation.py

```diff
@@ -253,24 +229,33 @@ def find_coordinate(*args, **kwargs):
     return find_coordinate
 
 
-def _get_interpolator(data_name, axis_names, map_coordinates_kwargs=None):
+DefaultMapCoordinatesOptions: MapCoordinatesOptions = {
```

Comment:
It always scares me to set default arguments deep in the code base. Especially the case here, since it is not innocuous at all (concerns what happens outside the grid, right?). Is there any reasonable place to do it close to the user-facing interface? Else, I would actually be in favour of requiring it as user input.